web/src/features/moreCast2/components/GridComponentRenderer.tsx

```diff
@@ -162,7 +162,7 @@ export class GridComponentRenderer {
     precision: number
   ) => {
     const oldValue = params.row[field].value
-    const newValue = params.value ? Number(params.value) : NaN
```

Comment:
When `params.value` had a value of 0 it erroneously evaluated to False and gave us back NaN instead of 0.